frontend/src/components/PopupWithForm/PopupWithForm.js

```diff
@@ -15,11 +13,19 @@ const PopupWithForm = (props) => {
           }}
           className="popup__block"
         >
+ <form
+              className="popup__form"
+              onSubmit={props.onSubmit}
+              action="#"
+              method="POST"
+             // name={"signup"}
+            >
           <button onClick={props.onClose} className="popup__close-btn"></button>
```

Comment:
`          <button onClick={props.onClose} className="popup__close-btn"></button>
`

when I close popups with the `cross buttons` there is a `submit` action. You need to add ` type="button"` to any buttons that are **not** submit buttons


```jsx

          <button
            type="button"
            onClick={props.onClose}
            className="popup__close-btn"
          />


```